### PR TITLE
[Selenium] Rework selenium tests in the "preferences", "autocomplete", "projectexplorer", "dependencies", "editor" packages for adopting to branch changes

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Rework selenium tests in the "preferences", "autocomplete", "projectexplorer", "dependencies", "editor" packages for adopting to branch changes

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/11609

### CODE REVIEW

**Review by artaleks9 — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/preferences/CheckErrorsWarningsTabTest.java [hunk 1]
```diff
@@ -96,8 +104,11 @@ public void errorsWarningTest() throws Exception {
     Assert.assertTrue(editor.getMarkersQuantity(WARNING_OVERVIEW) >= 12);
     Assert.assertEquals(editor.getMarkersQuantity(WARNING), 22);
 
+    editor.waitAnnotationsAreNotPresent(ERROR_OVERVIEW);
     menu.runCommand(TestMenuCommandsConstants.Profile.PROFILE_MENU, PREFERENCES);
     changeAllSettingsInErrorsWarningsTab(Preferences.DropDownValueForErrorWaitingWidget.ERROR);
+    Assert.assertEquals(editor.getMarkersQuantity(ERROR_OVERVIEW), 12);
+    Assert.assertEquals(editor.getMarkersQuantity(ERROR), 22);
```
Comment (musienko-maxim) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/preferences/CheckErrorsWarningsTabTest.java]: Static import?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/preferences/CheckErrorsWarningsTabTest.java [hunk 2]
```diff
@@ -127,6 +141,13 @@ private void changeAllSettingsInErrorsWarningsTab(
     WaitUtils.sleepQuietly(DEFAULT_TIMEOUT);
   }
 
+  private String getTextInStringFromFile(String path) throws Exception {
+    URL pathInUrl = CheckErrorsWarningsTabTest.class.getResource(path);
+    List<String> textFromFile =
+        Files.readAllLines(Paths.get(pathInUrl.toURI()), Charset.forName("UTF-8"));
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/preferences/CheckErrorsWarningsTabTest.java]:
```suggestion
        Files.readAllLines(Paths.get(pathInUrl.toURI()), StandardCharsets.UTF_8);
```

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/preferences/CheckErrorsWarningsTabTest.java [hunk 3]
```diff
@@ -127,6 +141,13 @@ private void changeAllSettingsInErrorsWarningsTab(
     WaitUtils.sleepQuietly(DEFAULT_TIMEOUT);
   }
 
+  private String getTextInStringFromFile(String path) throws Exception {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/preferences/CheckErrorsWarningsTabTest.java]:
```suggestion
  private String readFileToString(String pathToFile) throws IOException {
```

Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/preferences/CheckErrorsWarningsTabTest.java]: This method partly duplicates [FileUtil.readFileToString(File file)](https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FileUtils.html#readFileToString(java.io.File)).
It is also more suitable for [FileUtil](https://github.com/eclipse/che/blob/master/selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/utils/FileUtil.java#L21) class.

**Review by dmytro-ndp — APPROVED** [review]

**Review by musienko-maxim — APPROVED** [review]

**Pull request merged**